backend/sensehel/settings.py

```diff
@@ -85,7 +84,13 @@
         'NAME': os.getenv('POSTGRES_DB') or 'forum',
     }
 }
-
+if 'test' in sys.argv:
```

Comment:
This is not the idiomatic way to define database for tests. Check http://www.obeythetestinggoat.com/speeding-up-django-unit-tests-with-sqlite-keepdb-and-devshm.html

Reply:
Hi, the method represented will get me into some trouble that I'll have to run the test through Docker instead of the local machine since I'm using Mac. Indeed the testdb was intended for local use only, in production I'd like to use postgres for unit test though it's slower.

Reply:
OK. Let's merge and re-evaluate later.